core/Db/TransactionLevel.php

```diff
@@ -37,12 +40,15 @@ public function canLikelySetTransactionLevel()
 
     public function setUncommitted()
     {
+        $backup = $this->db->fetchOne('SELECT @@TX_ISOLATION');
```

Comment:
@diosmosis does this need to be `$this->statusBackup` instead of `$backup`?

Reply:
That's done below after successfully changing it. If it fails for some reason to set it then we don't need to set it back.

Reply:
makes sense 👍